THIRD-PARTY-LICENSES

```diff
@@ -1,6 +1,6 @@
 
 
-** opentelemetry-python; version 0.43b0 -- https://github.com/open-telemetry/opentelemetry-python
+** opentelemetry-python; version 1.22.0 -- https://github.com/open-telemetry/opentelemetry-python
```

Comment:
I think this should have been 1.22.0 since the opentelemetry-python repo components are 1.x versioned.